src/sentry/api/endpoints/organization_details.py

```diff
@@ -346,50 +346,52 @@ def save(self):
         if not hasattr(org, "__data"):
             update_tracked_data(org)
 
+        data = self.validated_data
```

Comment:
Do you need to reassign `validated_data` to `data`. 

Not a blocker for merge.

Reply:
Not really. 

I did it so that it's shorter as a variable name, and to centralize it so there's only one spot to change it for the rest of the function.